packages/query/src/index.ts

```diff
@@ -945,11 +977,15 @@ const generateQueryHook = async (
     ];
 
     const queryKeyFnName = camel(`get-${operationName}-queryKey`);
-    const queryKeyProps = toObjectString(
+    let queryKeyProps = toObjectString(
       props.filter((prop) => prop.type !== GetterPropType.HEADER),
       'implementation',
     );
 
+    if (OutputClient.VUE_QUERY) {
```

Comment:
@anymaniax is`OutputClient.VUE_QUERY` a reliable way to detect if the runner is vue only? 

Ta